### PR TITLE
Add "What are Docker Configs?" SEO Page

### PR DESCRIPTION
**Pull request by toriancrane**
## Description

This PR adds the content for the "Docker Configs" SEO page.

URL path: `/what-is/what-are-docker-configs/`

Resolves [this issue](https://github.com/pulumi/marketing/issues/837).

### CODE REVIEW

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: What are Docker Configs?
+meta_desc: |
+     Learn more about what Docker Configs are and how to use them.
+
+type: what-is
+page_title: "What are Docker Configs"
+
+customer_logos:
```
Comment (desteves) [inline themes/default/content/what-is/what-are-docker-configs.md:9]: Feel free to remove these.

**Review by desteves — APPROVED** [review]
LGTM

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+Docker, a leading platform in containerization technology, has revolutionized how applications are developed, shipped, and deployed. An essential facet of this ecosystem is the effective management of configuration data. [Docker Configs](https://docs.docker.com/engine/swarm/configs/) is a feature specially crafted to handle non-sensitive configuration information within Docker environments. This guide explores the ins and outs of Docker Secrets, highlighting its importance, functionality, and best practices.
```
Comment (aaronkao) [inline themes/default/content/what-is/what-are-docker-configs.md:32]:
```suggestion
Docker, a leading platform in containerization technology, has revolutionized how applications are developed, shipped, and deployed. An essential facet of this ecosystem is the effective management of configuration data. [Docker Configs](https://docs.docker.com/engine/swarm/configs/) is a feature specially crafted to handle non-sensitive configuration information within Docker environments. This guide explores the ins and outs of Docker Configs, highlighting its importance, functionality, and best practices.
```

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+### What are Docker Configs?
+
+Docker Configs are a resource in Docker for storing non-sensitive information such as configuration files, separate from a service's image or running containers. This enables keeping Docker images as generic as possible without the need for bind-mounting configuration files into containers or using environment variables. Unlike Docker Secrets, Docker Configs are not encrypted at rest and are directly mounted into the container's filesystem.
```
Comment (aaronkao) [inline themes/default/content/what-is/what-are-docker-configs.md:36]: hyperlink to Docker Secrets

then add this same phrasing in the Docker Secrets page that references and hyperlinks back to Docker Configs

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+#### Key Features
+
+- **Separation of Configuration from Code**: Docker Configs allow you to store configuration files outside of your Docker images, leading to more generic and reusable images.
+- **Flexibility in Management**: Configs can be added, updated, or removed from services dynamically, without the need to rebuild or restart containers.
+- **Secure Transmission**: Configs are sent to the swarm manager over a mutual TLS connection and are stored securely.
```
Comment (aaronkao) [inline themes/default/content/what-is/what-are-docker-configs.md:42]: is swarm manager a proper noun?

File: themes/default/content/what-is/what-are-docker-configs.md (hunk 2 above)
Comment (interurban) [inline themes/default/content/what-is/what-are-docker-configs.md:32]: We call out best practices as a topic; but are missing any below

**Comment by toriancrane** [comment]
For both this and Secrets page, mention a few best practices.

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+- **Easy Access within Containers**: Configs are automatically mounted into the container's filesystem, making them easily accessible to applications.
+- **Support for Various Data Types**: Configs can store generic strings or binary content, providing flexibility for different types of configuration data.
+
+### Creating Configs
+
+Docker Configs can be created via the Docker CLI.
```
Comment (aaronkao) [inline themes/default/content/what-is/what-are-docker-configs.md:48]: Is there another sentence that we want add here? In your previous Secrets one, you didn't use a h4 for install docker

File: themes/default/content/what-is/what-are-docker-configs.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+docker --version
+```
+
+{{< notes type="info" >}}
+
+The command to start Docker depends on your operating system. The above commands show examples for how to do this on Linux. You can find the commands relevant to your own operating system in the [Docker documentation](https://docs.docker.com/engine/install/).
+
+{{< /notes >}}
+
+##### Create a Swarm
```
Comment (aaronkao) [inline themes/default/content/what-is/what-are-docker-configs.md:79]: it seems like configs only matter with swarm but in the earlier paragraphs you never mention that fact. something to maybe elaborate

Comment (toriancrane) [reply themes/default/content/what-is/what-are-docker-configs.md:79]: It is included in the "prerequisites" section when talking about initializing a swarm and also in the considerations/challenges section.

File: themes/default/content/what-is/what-are-docker-configs.md (hunk 5 above)
Comment (toriancrane) [reply themes/default/content/what-is/what-are-docker-configs.md:48]: I will update this so it looks less weird.

**Pull request merged**